infra/app/build-repository/main.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  project_name = "<PROJECT_NAME>"
+  app_name     = "<APP_NAME>"
+  region       = "<REGION>"
```

Comment:
If region will be the same for every app we could create a "default_region" in `module/common/ouputs.tf`

Reply:
Ok, I'm beginning to understand what `modules/common` is for. What do you think about my comment in the other thread about potentially renaming / moving the `common` module to a place that's more visible to developers

Reply:
I think pulling common out of module moving it up to infra/x makes sense. I wonder if we should rename common to something more intuitive. e.g. constants or project-confg as you mentioned above.

Reply:
sounds good, i'll make those changes